src/endpoints/Campaigns/Client.ts

```diff
@@ -56,6 +56,15 @@ export function createClient(api: DeferredJobsApiClient) {
         return api.delete(`${routing.campaignsUrl}/${id}`);
     }
 
+    async function bulkDelete(
+        payload: BulkDeletePayload,
+    ): ProgressPromise<{ records_deleted_number: number }> {
```

Comment:
Is `ProgressPromise` the correct return type here if the API just returns JSON with `records_deleted_number`?

Reply:
I can't say what will future requirements bring, I can only say that right now we delete immediately.

Even for non drafts this implementation should stay the same, but in case we notice some performance issues we will need to rethink it.

I don't think we should worry about that now.